### ISSUE TITLE
libusb1-glue.c compile: gcc outputs "may be used uninitialized" warnings

### ISSUE THREAD

**Issue by noell** [libmtp/libmtp#137]
Compiling libmtp-1.1.20 with gcc (Debian 12.2.0-3), libusb1-glue.c compile outputs "may be used uninitialized" warnings.

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I/usr/local/include/libusb-1.0 -g -O2 -Wall -Wmissing-prototypes -MT libmtp_la-ptp.lo -MD -MP -MF .deps/libmtp_la-ptp.Tpo -c ptp.c -o libmtp_la-ptp.o >/dev/null 2>&1
mv -f .deps/libmtp_la-ptp.Tpo .deps/libmtp_la-ptp.Plo
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..    -I/usr/local/include/libusb-1.0 -g -O2 -Wall -Wmissing-prototypes -MT libmtp_la-libusb1-glue.lo -MD -MP -MF .deps/libmtp_la-libusb1-glue.Tpo -c -o libmtp_la-libusb1-glue.lo `test -f 'libusb1-glue.c' || echo './'`libusb1-glue.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I/usr/local/include/libusb-1.0 -g -O2 -Wall -Wmissing-prototypes -MT libmtp_la-libusb1-glue.lo -MD -MP -MF .deps/libmtp_la-libusb1-glue.Tpo -c libusb1-glue.c  -fPIC -DPIC -o .libs/libmtp_la-libusb1-glue.o
In function ‘ptp_exit_send_memory_handler’,
    inlined from ‘ptp_usb_senddata’ at libusb1-glue.c:1347:2:
libusb1-glue.c:1237:31: warning: ‘memhandler.priv’ may be used uninitialized [-Wmaybe-uninitialized]
 1237 |         PTPMemHandlerPrivate* priv = (PTPMemHandlerPrivate*)handler->priv;
      |                               ^~~~
libusb1-glue.c: In function ‘ptp_usb_senddata’:
libusb1-glue.c:1311:24: note: ‘memhandler’ declared here
 1311 |         PTPDataHandler memhandler;
      |                        ^~~~~~~~~~
In function ‘ptp_exit_send_memory_handler’,
    inlined from ‘ptp_usb_sendreq’ at libusb1-glue.c:1288:2:
libusb1-glue.c:1237:31: warning: ‘memhandler.priv’ may be used uninitialized [-Wmaybe-uninitialized]
 1237 |         PTPMemHandlerPrivate* priv = (PTPMemHandlerPrivate*)handler->priv;
      |                               ^~~~
libusb1-glue.c: In function ‘ptp_usb_sendreq’:
libusb1-glue.c:1262:25: note: ‘memhandler’ declared here
 1262 |         PTPDataHandler  memhandler;
 ```

**Comment by msmeissn** [comment]
should  be fixed by commit c4aa42c161474d03d3ce9137942aa0d8d58ba46f already

**Comment by noell** [comment]
confirmed, synced up to https://github.com/libmtp/libmtp/commit/c4aa42c161474d03d3ce9137942aa0d8d58ba46f and rebuilt, no warning anymore.
